docs/chapters/sdk/using-the-sdk-to-crosscompile.rst

```diff
@@ -0,0 +1,21 @@
+Using the SDK to cross compile
+==============================
+
+Cross compiling with help of the SDK is not much different to normal compiling, there are more pitfalls but let's get on with the happy cases for now.
+
+Sourcing Environment
+--------------------
+
+Once you have installed the SDK you need to source it, which means that the all the right environment variables will be set in your terminal session so the right compiler and sysroot will be used during the compilation.
```

Comment:
... which means that the SDK toolchain and sysroot environment variables will be set in your active terminal session.